roles/configure/templates/bastion-cloud-config.yml.j2

```diff
@@ -52,6 +52,13 @@ write_files:
     wget https://storage.googleapis.com/kubernetes-release/release/v1.2.2/bin/linux/amd64/kubectl -O /tmp/kubectl
     chmod a+x /tmp/kubectl
     sudo mv /tmp/kubectl /usr/local/bin/kubectl
-
+    output={{ version.results[0].stdout }}
+    size=${{ '{' }}#output{{ '}' }}
     git clone https://github.com/bcwaldon/klondike /home/ubuntu/klondike
+    cd /home/ubuntu/klondike
+    if [ $size == 40 ]; then
```

Comment:
Since the magic number 40 isn't immediately obvious, could you add a comment above this line?